src/commands/publish.ts

```diff
@@ -206,14 +206,21 @@ async function printRevisionSummary(
 /**
  * Prompt the user that everything is OK and we should proceed with publishing
  */
-async function promptConfirmation(): Promise<void> {
+async function promptConfirmation(targetList: BaseTarget[]): Promise<void> {
+  logger.info('Publishing to targets:');
+
+  logger.info(
+    targetList.map(target => `  - ${getTargetId(target.config)}`).join('\n')
+  );
+  logger.info(' ');
+
   if (hasInput()) {
     const questions = [
       {
         message: 'Is everything OK? Type "yes" to proceed:',
         name: 'readyToPublish',
         type: 'input',
-        validate: (input: string) => input.length > 2 || 'Please type "yes"',
+        validate: (input: string) => input.length >= 2 || 'Please type "yes"',
```

Comment:
I don’t understand this code. What is this supposed to do?

Reply:
Can add a comment :D